action_server/src/action_server/actions/find.py

```diff
@@ -27,7 +27,7 @@ def __init__(self):
         Action.__init__(self)
         # TODO: change this to a python dictionary schema
         self._required_field_prompts = {'object': " What exactly would you like me to find? "}
-        self._required_skills = ['head', 'base', 'rightArm', 'speech']
+        self._required_skills = ['head', 'base', 'arms', 'speech']  # ToDO: Still calls rightArm directly
```

Comment:
arms is set to be removed soon. So unless we want to break things again we should change it to _arms
In the long term we should try to make sure that we are not reliant on the exact name of the parts, only their class and properties.

Reply:
Also todo is no longer needed
```suggestion
        self._required_skills = ['head', 'base', '_arms', 'speech']
```

Reply:
```suggestion
        self._required_skills = ['head', 'base', 'speech']
```